.env.example

```
@@ -6,4 +6,9 @@ NEXT_PUBLIC_USAGE_DASHBOARD_ID=
 NEXT_PUBLIC_USAGE_DASHBOARD_DOMAIN=
 
 # Pendo Analytics
-NEXT_PUBLIC_PENDO_API_KEY=
\ No newline at end of file
+NEXT_PUBLIC_PENDO_API_KEY=
+
+# E2E Testing (Playwright)
+# Set these to run E2E tests that require authentication
+E2E_TEST_EMAIL=
+E2E_TEST_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)